app/views/api/v0/articles/show.json.jbuilder

```diff
@@ -22,8 +22,6 @@ json.last_comment_at    @article.last_comment_at&.utc&.iso8601
 
 json.body_html @article.processed_html
 json.body_markdown @article.body_markdown
-json.ltag_style(@article.liquid_tags_used.map { |ltag| Rails.application.assets["ltags/#{ltag}.css"].to_s.html_safe })
```

Comment:
these two properties, `ltag_style` and `ltag_script` are never used anywhere in the code either, they are neither documented. I think it's safe to remove them as well.

Reply:
nice catch!